get-gke-credentials/README.md

```diff
@@ -0,0 +1,119 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# get-gke-credentials
+
+This action configures authentication to a [GKE cluster][gke] via a `kubeconfig` file that can be used with `kubectl` or other methods of interacting with the cluster.
+
+Authentication is performed by generating a [short-lived token][token] (default behaviour) or via the [GCP auth plugin][gcp-auth-plugin] present in `kubectl` which uses the service account keyfile path in [GOOGLE_APPLICATION_CREDENTIALS][gcp-gcloud-auth].
+
+## Prerequisites
+
+This action requires:
+
+- Google Cloud credentials that are authorized to view a GKE cluster. See the Authorization section below for more information.
+
+- [Create a GKE cluster](https://cloud.google.com/kubernetes-engine/docs/quickstart?_ga=2.267842766.1374248275.1591025444-475066991.1589991158)
+
+## Usage
+
+```yaml
+steps:
+- id: get-credentials
+  uses: GoogleCloudPlatform/github-actions/get-gke-credentials@master
+  with:
+    name: my-cluster
+    location: us-central1-a
+    credentials: ${{ secrets.gcp_credentials }}
+
+# The KUBECONFIG env var is automatically exported and picked up by kubectl.
+- id: get-pods
+  run: kubectl get pods
+```
+
+## Inputs
+
+- `name`: (Required) Name of the cluster to get credentials for.
+
+- `location`: (Required) Location (Region/Zone) for the cluster.
+
+- `credentials`: (Optional) Service account key to use for authentication. This should be
+  the JSON formatted private key which can be exported from the Cloud Console. The
+  value can be raw or base64-encoded. Required if not using a the
+  `setup-gcloud` action with exported credentials.
+
+- `project_id`: (Optional) Project id where the cluster is deployed. If provided, this
```

Comment:
```suggestion
- `project_id`: (Optional) Project ID where the cluster is deployed. If provided, this
```

Reply:
add note about if not provided

Reply:
@averikitsch I have made this  description similar to our other actions. project_id is expected for this action either via setup-gcloud or provided via this input.